code/__DEFINES/~pariah_defines/DNA.dm

```diff
@@ -17,3 +17,9 @@
 
 #define MANDATORY_FEATURE_LIST list("flavor_text" = "")
 
+//Species
+#define DNA_TAJARAN_MARKINGS_BLOCK 17
+#define DNA_TAJARAN_TAIL_BLOCK 18
+#define DNA_TAJARAN_SNOUT_BLOCK 19
```

Comment:
dont modularize these, these need to be where everything else is so people dont add dna blocks with the same values